app/backend/app.py

```diff
@@ -84,10 +84,26 @@ async def assets(path):
 
 
 # Serve content files from blob storage from within the app to keep the example self-contained.
-# *** NOTE *** this assumes that the content files are public, or at least that all users of the app
-# can access all the files. This is also slow and memory hungry.
+# *** NOTE *** if AZURE_ENFORCE_ACCESS_CONTROL is not set or false, all content is accessible
```

Comment:
To clarify, if they enforce authentication (but not access control), won't easyauth kick out non-logged in users with an HTTP 401 before returning this route?

Reply:
Correct, but easy-auth is auth only, it doesn't understand authorization (who can access what document). I can update the comment